lib/postProcessing/include.js

```diff
@@ -7,6 +7,7 @@ var externalRequest = require("request").defaults({
   pool: { maxSockets: Infinity }
 });
 var async = require("async");
+var includeDebug = require("debug")("jsonApi:include");
```

Comment:
What do you think about doing these like:
```
var debug = {
  include: require("debug")("jsonApi:include")
};
```
which will make it nicely namespaced if we use more than one in the same file?